packages/peregrine/lib/talons/Region/useRegion.js

```diff
@@ -48,11 +48,12 @@ export const useRegion = props => {
     }, [country, regionInputFieldApi, regionSelectFieldApi]);
 
     const { data, error, loading } = useQuery(getRegionsQuery, {
-        variables: { countryCode: country }
+        variables: { countryCode: country },
+        skip: !country
     });
 
     let formattedRegionsData = [{ label: 'Loading Regions...', value: '' }];
-    if (!loading && !error) {
+    if (data && !loading && !error) {
```

Comment:
Some older code of mine before realizing we only care about the truthy-ness of `data`; this can probably be simplified:

```suggestion
    if (data) {
```